sormas-e2e-tests/src/main/java/org/sormas/e2etests/pages/application/cases/EditCasePage.java

```diff
@@ -231,6 +232,8 @@ public class EditCasePage {
   public static final By ARCHIVE_CASE_BUTTON = By.id("archiveDearchive");
   public static final By ARCHIVE_RELATED_CONTACTS_CHECKBOX =
       By.cssSelector(".popupContent span[class='v-checkbox v-widget']");
+  public static final By INFRASTRUCTURE_DATA_POPUP = By.cssSelector("#gwt-uid-662");
```

Comment:
This ID might change after reloading a page or when opening it with different browser. I'd advise using something more persistent (the popup text, for instance).